cita-chain/types/src/block_number.rs

```diff
@@ -0,0 +1,30 @@
+use cita_types::H256;
+use jsonrpc_types::rpc_types::{BlockNumber as RpcBlockNumber, BlockTag as RpcBlockTag};
+
+pub type TransactionHash = H256;
+pub type BlockNumber = u64;
+
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub enum BlockTag {
```

Comment:
I feel so strange about this define!
Do you think `Height` and `Hash` is a `Tag`?
If `yes`, it should be included in `Tag` enum.
If `not`, the enum name should not be `BlockTag`.

Reply:
I don't have a better idea for now.

Reply:
At present, just let it go. More modified later.